### PR TITLE
Copy download-or-bust helper from gce to aws

### PR DESCRIPTION
**Pull request by justinsb**
During a push, we were wget-ing to <filename>.1, .2, etc, which wasn't working

### CODE REVIEW

**Comment by justinsb** [comment]
Spotted the ```echo "Downloading file ($SERVER_BINARY_TAR_URL)"``` in the helper function, which I copied from GCE.  It should be ```Downloading file $1``` I think.  Should I fix here or in another PR?

**Comment by roberthbailey** [comment]
I'd fix the helper function in this PR.

**Comment by justinsb** [comment]
OK - thanks: Fixed the message in the helper function as well.

File: cluster/aws/templates/common.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
```
Comment (roberthbailey) [inline cluster/aws/templates/common.sh]: nit: 2015

File: cluster/aws/util.sh [hunk 2]
```diff
@@ -601,6 +602,38 @@ function kube-down {
   fi
 }
 
+# Update a kubernetes cluster with latest source
+function kube-push {
+  #detect-project
```
Comment (roberthbailey) [inline cluster/aws/util.sh]: It's better to delete this line rather than comment it out

**Comment by roberthbailey** [comment]
Couple of nits; otherwise LGTM.

**Comment by justinsb** [comment]
Don't merge yet ... I somehow touched the previous commit during my squash

**Comment by justinsb** [comment]
OK, fixed up the git problems.  Sorry about that.

I made the proposed changes: fixed the copyright and removed the commented out detect-project.

**Comment by roberthbailey** [comment]
LGTM.

**Comment by roberthbailey** [comment]
Waiting for travis to pass before merging.

**Comment by roberthbailey** [comment]
Poking travis again....

**Pull request merged**